docs/fastdds/logging/logging_configuration.rst

```diff
@@ -0,0 +1,214 @@
+filter are consumed (see :ref:`dds_layer_log_filter`).
+This component cannot be further configured or disabled.
+
+
+.. _dds_layer_log_file_context:
+
+File Context
+^^^^^^^^^^^^
+
+This component specifies the origin of the log entry in terms of file name and line number (see
+:ref:`dds_layer_log_logging` for a log entry example featuring this component).
+This is useful when tracing code flow for debugging purposes.
+The file context component can be enabled/disabled using the member function |Log::ReportFilenames|.
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+    :language: c++
+    :start-after: //LOG_REPORT_FILENAMES
+    :end-before: //!--
+    :dedent: 4
+
+
+.. _dds_layer_log_function_name:
+
+Function Name
+^^^^^^^^^^^^^
+
+This component specifies the origin of the log entry in terms of the function name (see
+:ref:`dds_layer_log_logging` for a log entry example featuring this component).
+This is useful when tracing code flow for debugging purposes.
+The function name component can be enabled/disabled using the member function |Log::ReportFunctions|.
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+    :language: c++
+    :start-after: //LOG_REPORT_FUNCTIONS
+    :end-before: //!--
+    :dedent: 4
+
+
+.. _dds_layer_log_register_consumers:
+
+Register Consumers
+^^^^^^^^^^^^^^^^^^
+
+*eProsima Fast DDS* logging module supports zero or :ref:`consumers <dds_layer_log_consumer>` logging the entries
+registered in the logging queue with the methods described in :ref:`dds_layer_log_logging`.
+To register a consumer, the |Log| class exposes member function |Log::RegisterConsumer|
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+    :language: c++
+    :start-after: //LOG_REGISTER_CONSUMER
+    :end-before: //!--
+    :dedent: 4
+
+The consumers list can be emptied with member function |Log::ClearConsumers|.
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+    :language: c++
+    :start-after: //LOG_CLEAR_CONSUMERS
+    :end-before: //!--
+    :dedent: 4
+
+.. note::
+
+    Registering and configuring consumers can also be done using *Fast DDS* XML configuration files.
+    Please refer to :ref:`dds_layer_log_xml` for details.
+
+.. warning::
+
+    |Log::ClearConsumers| empties the consumers lists.
+    All log entries are discarded until a new consumer is register via |Log::RegisterConsumer|, or until |Log::Reset| is
+    called.
+
+.. _dds_layer_log_reset:
+
+Reset Configuration
+^^^^^^^^^^^^^^^^^^^
+
+The logging module's configuration can be reset to default settings with member function |Log::Reset|.
+
+.. warning::
+
+    Resetting the module's configuration entails:
+
+    * Setting :ref:`dds_layer_log_verbosity_level` to |Log::Kind::Error|.
+    * Disabling :ref:`dds_layer_log_file_context` component.
+    * Enabling :ref:`dds_layer_log_function_name` component.
+    * Clear all :ref:`dds_layer_log_filter`.
+    * Clear all consumers and set a STDOUT consumer.
+
+
+.. _dds_layer_log_xml:
+
+XML Configuration
```

Comment:
All this has already been moved to the XML profile section. Change to something like this:
https://fast-dds.docs.eprosima.com/en/latest/fastdds/dynamic_types/xml_profiles.html

Reply:
I suggest you do that on the XML profiles PR. It's because that way we'll have the cross-reference

Reply:
I took care of it